ui/ui-components/components/session/SignIn.tsx

```diff
@@ -53,7 +53,7 @@ export default function SignInForm(props) {
     setLoadingOauthProviders(true);
     const response: Actions.OAuthListProviders = await execApi(
       "get",
-      `/oauth/providers`
+      `/oauth/providers?type=user`
```

Comment:
Note that until this is merged in, if we merge the telemetry counterpart of this PR that adds the mailchimp provider, it will start showing up on the sign in page.